### DISCUSSION TITLE
make a offline html version

### DISCUSSION THREAD

**Discussion by DaMonkeExe** [ruffle-rs/ruffle#6306] [Ideas]
is it possible to make a offline html version of ruffle to run those old flash games offline in a web browser ?

**Comment by wearyexcellance** [comment]
That's exactly what I'm trying to do. Not sure how but I'm trying to change the ruffle.js code

**Reply by DaMonkeExe** [reply]
i dont think changing the code would help i dont think .wasm files load on a web browser locally

**Reply by fadedjump** [reply]
It actually can be; I created an html that converts huge files to there base64 format (like 2+ gb files) so it would be possible; the only problem that I have is modifying the javascript code to reference it to the base64 wasm; I mean its possible, but every single reference to the code and modify it so accept it within the code... way too many references. I'm happy to help though if someone else wants to help as well.

**Reply by SigwardWaters** [reply]
fadedjump to the rescue.. and then there's Fable 5.

**Comment by n0samu** [comment]
This isn't exactly what you're asking for, but I think it's the closest thing possible: https://github.com/jooy2/open-ruffle-player/

**Reply by HedgyhogGuy** [reply]
what about Chromebooks? isn't there an HTML file of this?

**Comment by KingNullboy** [comment]
I DID IT
IT TOOK ME FOREVER
BUT I DID IT
[Ruffle.html](https://github.com/user-attachments/files/26636293/Ruffle.html)

**Reply by minecrafter-124** [reply]
> I DID IT IT TOOK ME FOREVER BUT I DID IT [Ruffle.html](https://github.com/user-attachments/files/26636293/Ruffle.html)

THANK YOU!!! I WAS TRYING TO FIND SOMETHING LIKE THIS FOR THE PAST FEW DAYS!!!

**Reply by KingNullboy** [reply]
You're welcome! I probably needed it for a similar reason to you, too
